src/WebJobs.Script.WebHost/Models/HostStatus.cs

```diff
@@ -44,5 +44,11 @@ public class HostStatus
         /// </summary>
         [JsonProperty(PropertyName = "processUptime", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public long ProcessUptime { get; set; }
+
+        /// <summary>
+        /// Gets or sets the information related to Extension bundles
+        /// </summary>
+        [JsonProperty(PropertyName = "extensionBundle", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public ExtensionBundle ExtensionBundle { get; set; }
```

Comment:
nit: reaname to ExtensionBundleDetails  or ExtensionBundleInfo

Reply:
do you mean the type or the json property?

Reply:
Thinking about it a little mode ExtensionBundle feels more accurate, because while assigning or consuming the property adding something like `ExtensionBundleDetails.id` feels odd.